packages/lexicon/src/serialize.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  cborDecode,
+  cborEncode,
+  check,
+  cidForCbor,
+  dataToCborBlock,
+  IpldValue,
+  ipldToJson,
+  jsonToIpld,
+  JsonValue,
+  schema,
+} from '@atproto/common'
+import { BlobRef, jsonBlobRef } from './blob-refs'
+
+export type LexValue =
+  | IpldValue
+  | BlobRef
+  | Array<LexValue>
+  | { [key: string]: LexValue }
+  | { [key: number]: LexValue }
```

Comment:
I'm trying to think of how this type is possible since keys are generally always strings.  But realize I may be / probably am missing something!

Reply:
I honestly thought js objects could have numbers as keys but after a quick look, turns out those are converted to strings under the hood 😅 

I'll do away with this 👍